src/js/dataEntry.js

```diff
@@ -0,0 +1,84 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("gpii.chartAuthoring.dataEntry", {
+        gradeNames: ["gpii.chartAuthoring.templateInjection", "gpii.chartAuthoring.valueBinding", "autoInit"],
+        selectors: {
+            input: ".gpiic-ca-dataEntry-value",
+            percentage: ".gpiic-ca-dataEntry-percentage",
+            label: ".gpiic-ca-dataEntry-label"
+        },
+        strings: {
+            inputPlaceholder: "Value",
+            percentage: "%percentage%"
+        },
+        model: {
+            value: null,
+            label: "",
+            perecentage: null,
+            total: null
+        },
+        modelRelay: [{
+            source: "value",
+            target: "value",
+            singleTransform: {
+                type: "gpii.chartAuthoring.transforms.stringToNumber"
+            }
+        },{
+            source: "",
+            target: "percentage",
+            singleTransform: {
+                type: "gpii.chartAuthoring.transforms.percentage",
+                value: "{that}.model.value",
+                total: "{that}.model.total"
+            }
+        }],
+        bindings: {
+            input: "value",
+            label: "label"
+        },
+        invokers: {
+            setPercentage: {
+                funcName: "gpii.chartAuthoring.percentage.render",
+                args: [
+                    "{that}.dom.percentage",
+                    "{that}.model.percentage",
+                    "{that}.options.strings.percentage",
+                    2
```

Comment:
This should be renamed something more descriptive, such as "updatePercentage" or "refreshPercentage", and the literal number 2 here needs to be broken out into a component option